keyboards/handwired/scottokeebs/scottowing/info.json

```diff
@@ -0,0 +1,85 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoWing",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "matrix_pins": {
+        "cols": [
+    "GP14", "GP29", "GP28", "GP27", "GP26", "GP15", "GP7", "GP6", "GP5", "GP4", "GP3", "GP8"
+        ],
+        "rows": ["GP9", "GP10", "GP11", "GP12"]
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x1020",
+        "vid": "0x534B"
+    },
+    "ws2812": {
+        "pin": "GP16",
+        "driver": "vendor"
+    },
+    "rgblight": {
+        "led_count": 1
+    },
+    "layouts": {
```

Comment:
@joe-scotto 
Quick check: would there be purpose in adding a `split_3x5_2` layout (i.e. 36 keys) for this board?

Reply:
@dunk2k I thought about it but since it's handwired it's simpler to keep it as 3x6 and then just leave off pin 14 / 8 if building the 3x5.